fix(swabbie): Update swabbie email tempalte to link ASG resource

aravindmd opened the pull request:
- Update the email template to include link to the resource on spinnaker web ui , `additionalContext` will include `spinnakerWebUrl` which is used to construct the `resourceUrl`. Since we are not using email notifications for any other swabbie resource i just updated the template without any conditional checks.

aravindmd commented:
<img width="1214" alt="Screen Shot 2019-07-08 at 2 45 15 PM" src="https://user-images.githubusercontent.com/21320508/60844789-39956c80-a18f-11e9-9f17-83169d11b1bd.png">

jeyrschabu commented:
@aravindmd thanks for adding that screenshot. I will defer to @gcomstock for comments on the look and feel. Also might help to see a screenshot with more than one resource.

A couple of things jump out, I notice some labels starting in Cap and others are lowercase. I think the `link` and `id` could be combined to point to the resource in Spinnaker.

jeyrschabu commented on echo-notifications/src/main/resources/templates/swabbie/body.ftl:
```diff
@@ -138,6 +139,17 @@
+              <td class="loop__key" align="right" valign="top" style="padding: 4px 16px 4px 0; font-family: Helvetica, Arial, sans-serif; font-size: 12px; font-weight: bold;" width="30%">
+                Link
+              </td>
+              <td align="left" style="padding: 4px 0; font-family: Helvetica, Arial, sans-serif; font-size: 12px;">
```
Per our chat, lets remove the specific link. In the notification context in swabbie, lets pass in a `resourceType` and a `resourceUrl` (iirc this field should be there already) i.e `resourceUrl: https://spinnaker/search?q=`

https://github.com/spinnaker/swabbie/blob/3440531305926e9cda41cd5b9e93445941e701dc/swabbie-core/src/main/kotlin/com/netflix/spinnaker/config/SwabbieProperties.kt#L92

aravindmd replied:
Agree . I changed the template to use the `spinnakerWebUrl` ( i think we still need this, its different from `resourceUrl` which links to the api route and not the web ui ) and use the `resourceType` we already have as part of the `notificationContext`

jeyrschabu replied:
`spinnakerweburl` is what `resourceUrl` was intended to be. It's not meant to be for api but for visualizing the resource in spinnnaker UI

aravindmd commented:
<img width="1212" alt="Screen Shot 2019-07-08 at 4 11 24 PM" src="https://user-images.githubusercontent.com/21320508/60849288-eaefce80-a19e-11e9-8681-14685160a789.png">

gcomstock commented:
+1 to Jeyrs' comment: just make the id clickable and drop the link key/val. If you could expand the button column just a bit so "Opt Out of Delete" worked on one line without wrapping, that'd tidy things up a bit too. Thanks!

aravindmd commented:
Updated the id to make it clickable and removed the link row , using `spinnakerLink` from notification context and also update the column width to avoid line wrapping.

aravindmd commented:
<img width="1207" alt="Screen Shot 2019-07-09 at 9 38 49 AM" src="https://user-images.githubusercontent.com/21320508/60907035-6a2be380-a22d-11e9-888a-47d4adf8d4d1.png">

gcomstock commented:
Thanks captain! Looks great.

aravindmd commented:
@jeyrschabu let me know if its good to go now
